modules/fleet/src/javaRestTest/java/org/elasticsearch/fleet/FleetSystemIndexIT.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.fleet;
+
+import com.carrotsearch.randomizedtesting.annotations.Name;
+import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.common.xcontent.XContentHelper;
+import org.elasticsearch.common.xcontent.json.JsonXContent;
+import org.elasticsearch.test.rest.ESRestTestCase;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class FleetSystemIndexIT extends ESRestTestCase {
+
+    private final String indexName;
+
+    public FleetSystemIndexIT(@Name("indexName") String indexName) {
+        this.indexName = indexName;
+    }
+
+    @ParametersFactory
+    public static Iterable<Object[]> data() {
+        return Arrays.asList(
+            new Object[] { ".fleet-outputs" },
+            new Object[] { ".fleet-policies" },
+            new Object[] { ".fleet-policies-enrollment-keys" },
+            new Object[] { ".fleet-policies-inputs" },
+            new Object[] { ".fleet-agents" },
+            new Object[] { ".fleet-agents-checkins" }
+        );
+    }
+
+    public void testCreateIndex() throws IOException {
+        Request request = new Request("PUT", "/_fleet/" + indexName);
+        Response response = client().performRequest(request);
+        assertThat(response.getStatusLine().getStatusCode(), is(200));
```

Comment:
Do you specifically want to check for 200, or would this be OK?
```suggestion
        assertOK(response);
```

Reply:
These were 100% copy/pasted from the Kibana tests with `s/kibana/fleet`, `assertOK` is probably better.